README.MD

```diff
@@ -1,3 +1,62 @@
 # Playdate Runner
 
-A runner style game created for the Playdate.
+A runner style game created for the Playdate. This is intended as a proof of concept to show the potential of Playdate development in the club.
+
+## How to Play
+
+Avoid obstacles for as long as possible. Press Up to jump, Down to duck, A to shoot, and B to pause the game.
```

Comment:
it might be worth it to include that scores can be reset on the main menu

Reply:
I agree, that would not be immediately evident to someone new to the game